core/src/main/java/hudson/console/ModelHyperlinkNote.java

```diff
@@ -27,7 +27,7 @@ public ModelHyperlinkNote(String url, int length) {
 
     @Override
     protected String extraAttributes() {
-        return " class='model-link'";
+        return " class='model-link model-link--float'";
```

Comment:
I thought I had mentioned it before (and may have, elsewhere), but stuff like this is a poor solution, as it will only apply to links generated after this update has been applied. Historical build logs, which are very important for many Jenkins users, will not get this change applied.